decidim-admin/app/helpers/decidim/admin/imports_helper.rb

```diff
@@ -12,8 +12,8 @@ module ImportsHelper
       # resource_id - The resource id that is passed to route.
       #
       # Returns a rendered dropdown.
-      def import_dropdown(component = current_component, resource_id = nil)
-        locals = { component: component, resource_id: resource_id }
+      def import_dropdown(component = current_component, options = {})
```

Comment:
I think we could change this to keyword argument as in:
```ruby
def import_dropdown(component = current_component, resource_id: nil)
```

Reply:
Yes